json-schema/schema.json

```diff
@@ -115,6 +115,16 @@
               }
             ]
           }
+        },
+       "cube:variables": {
+          "type": "object",
+          "additionalProperties": {
```

Comment:
The anyOf doesn't make much sense. Just put the $ref directly into additionalProperties.